.github/workflows/build-test-release.yml

```diff
@@ -4,8 +4,6 @@ on:
   push:
     branches:
       - main
-    tags:
-      - "*"
```

Comment:
I don't know why this would be changed in this PR.

Reply:
I don't exactly remember. I think we had an issue with the tag name not passing if those lines where included. I believe @dasconnor made the suggestion to me

Reply:
My guess here is that it was to avoid goreleaser from performing a release when the tag was made

Reply:
But then we can't publish a release by tag? Is this the desired behavior?

Reply:
PM with @dasconnor give no conclusion on the origin of this haha including it again